### ISSUE TITLE
Testing against SciPy to assess relative performance

### ISSUE THREAD

**Issue by pkofod** [JuliaOpt/LsqFit.jl#21]
Shameless repost of https://github.com/JuliaOpt/Optim.jl/issues/298

@ChrisRackauckas wrote:
*Are there any tests against the SciPy LM implementation to see if this implementation is as robust as possible (assuming the SciPy implementation is good)? I can't seem to find out how to properly use PyCall to use the SciPy curve_fit function, but I think it would be a necessary test since the current LM implementation is lacking a lot of robustness (and while that's a standard feature of the algorithm, it just seems even pickier than normal)*

**Comment by blakejohnson** [comment]
No, I haven't benchmarked against other implementations. Seems worthwhile, though.
